docs/integrations/sources/typeform.md

```diff
@@ -4,9 +4,22 @@ This page guides you through the process of setting up the Typeform source conne
 
 ## Prerequisites
 
-* token - The Typeform API key token.
-* start\_date - Date to start fetching Responses stream data from.
-* form_ids (Optional) - List of Form Ids to sync. If not passed - sync all account`s forms.
+- [Typeform Account](https://www.typeform.com/)
+- Start Date - Date to start fetching Responses stream data from.
+- form_ids (Optional) - List of Form Ids to sync. If not passed - sync all account`s forms.
```

Comment:
```suggestion
- Form IDs (Optional) - If you want to sync data for specific forms, you'll need to have the IDs of those forms. If you want to sync data for all forms in your account you don't need any IDs. 
```

Reply:
also, can you tell the user where to get the form IDs?

Reply:
added your suggestion, and added explanation about Form IDs